rasa_core/server.py

```diff
@@ -385,7 +385,7 @@ def predict(sender_id):
     @ensure_loaded_agent(agent)
     def log_message(sender_id):
         request_params = request.get_json(force=True)
-        message = request_params.get("text")
+        message = request_params.get("message")
```

Comment:
I don't think we can just change this.

Reply:
Hi @tmbo ! Thanks for reviewing this! Could you please provide more resons for this?

Reply:
well, it is a breaking api change as anyone who is using this api needs to change their payloads. I see the benefit of the change, making things more consistent - but it would be better to make this backwards compatible.

Reply:
Hi! I just added a "try except" handler for this!